src/Story.ts

```diff
@@ -43,20 +46,41 @@ class Story {
 		this.zoom = storyDataNode.getAttribute("zoom");
 		this.creator = storyDataNode.getAttribute("creator");
 		this.creatorVersion = storyDataNode.getAttribute("creator-version");
+		this.canvasMap = new CanvasMap();
 
 		const passages: Passage[] = [];
 		const passageNodes =
 			storyDataNode.querySelectorAll<HTMLElement>("tw-passagedata");
 
 		Array.from(passageNodes).forEach((node) => {
-			passages.push(new Passage(node));
+			const passage = new Passage(node);
+
+			if (passage.name === "MapDefaults") {
+				let defaultData: CanvasMapDefaultData;
+				try {
+					defaultData = JSON.parse(passage.rawContent);
+					this.canvasMap.setDefaultMapData(defaultData);
+				} catch (e) {
+					throw new Error(`Malformed default map data!`);
```

Comment:
TODO, throw the original error too? It's fairly clear what's here, but the original error would have the JSON context.